### PR TITLE
Delete single msgs

### LINKED ISSUE #181: allow to delete a single message
i think this should be done using a UIAlertController - which can then also be used for "info" or other options (do not show two controls as telegram does currently (one abottom of the screen, one beside the message))

### PR DESCRIPTION
**Pull request by cyBerta**
closes  #181

... but can be seen as a first iteration

### CODE REVIEW

File: deltachat-ios/Controller/ChatViewController.swift [hunk 1]
```diff
@@ -472,6 +481,20 @@ class ChatViewController: MessagesViewController {
             super.collectionView(collectionView, performAction: action, forItemAt: indexPath, withSender: sender)
         }
     }
+
+    private func askToDeleteMessage(id: Int) {
+        let alert = UIAlertController(title: String.localized("delete_message_ask"),
```
Comment (r10s) [inline deltachat-ios/Controller/ChatViewController.swift]: i think we should re-use `String.localizedStringWithFormat(String.localized("ask_delete_messages"), 1)`. already available in all languages, less work for translators and gives hint that the message is also deleted from the server.

**Comment by r10s** [comment]
apart from the hint above (and the conflict, of course :) this looks good to me and works as exected.

File: deltachat-ios/Controller/ChatViewController.swift (hunk 1 above)
Comment (cyBerta) [reply deltachat-ios/Controller/ChatViewController.swift]: fixed.

**Comment by r10s** [comment]
looks great! i think it is just fine using the nearby-tap-menu (how is it called?)

**Pull request merged**